integration

```diff
@@ -280,6 +280,8 @@
   "dmamontov/hass-seafile",
   "dmoranf/home-assistant-wattio",
   "dolezsa/thermal_comfort",
+  "domalab/ha-actronair-neo",
```

Comment:
Limit each PR to a single entry.
Your supplied links are not valid.

Reply:
Ok. I removed the domalab/ha-actronair-neo integration until my UNRAID one gets approved then I'll do the other one.

Reply:
If you look at the size of the queue, you might want to reconsider that and do both as separate now ;)
Hint: do not use the "master" branch of your fork..
https://hacs.xyz/docs/publish/include/#before-submitting